tests/integration_tests_plugins/dockercompute/operations.py

```diff
@@ -127,6 +127,7 @@ def _init_script_agent_setup(container_id, install_agent_script):
 
 def _remote_agent_setup(container_id):
     _wait_for_ssh_setup(container_id)
+    _docker_exec(container_id, 'mkdir -p /root/.ssh/')
```

Comment:
why is this needed?

Reply:
I suspect that in the new centos image, there's no `.ssh` dir. Over at Jenkins the image has it. Maybe over at Jenkins we have a custom centos image but the one you download from the internet doesn't have the dir.

Reply:
There was no update to the image in the recent time

Reply:
Then maybe some other program changed its behavior. The dir doesn't exist at that point, it fails without this command. Also this looks like a common command, saw another procedure over at `docl`. Your thoughts?

Reply:
it's not critical, but strange